src/diagnostics.rs

```diff
@@ -199,7 +199,10 @@ fn verible_syntax(
         .arg("-")
         .spawn()
         .ok()?;
-    let re = Regex::new(r"^.+:(?P<line>.*):(?P<col>.*):\s(?P<message>.*)\s.*$").ok()?;
+    let re = Regex::new(
```

Comment:
can you make this regex pre-compiled?
https://docs.rs/regex/latest/regex/#avoid-re-compiling-regexes-especially-in-a-loop